src/utils.js

```diff
@@ -1026,15 +1023,13 @@ Array.prototype.indexOf = Array.prototype.indexOf || function (el, offset) {
 };
 
 /**
-* Because our older API parameters are not consistent with other array API methods, and also
-* because only [IE8 lacks integrated support]{@glossary polyfill} for
-* [lastIndexOf()]{@glossary Array.lastIndexOf} we ensure it is defined (only IE8) and
-* advise, moving forward, that the built-in method be used. But to preserve our original API, it
-* will simply call this method, knowing it exists.
+* Because our older API parameters are not consistent with other array API methods, we advise,
+* moving forward, that the built-in method be used. But to preserve our original API, it will
+* simply call this method, knowing it exists.
 *
 * @private
 */
-Array.prototype.lastIndexOf = Array.prototype.lastIndexOf || function (el, offset) {
+Array.prototype.lastIndexOf = function (el, offset) {
```

Comment:
This is the wrong approach.  If we need only for IE8 then this whole thing can be removed.  Same for indexOf.  This overwrites the built-in with ours, which we absolutely don't want.